hyperlane/Dockerfile

```diff
@@ -30,5 +30,6 @@ WORKDIR /home/hyperlane
 COPY hyperlane/configs /home/hyperlane/configs
 COPY hyperlane/registry /home/hyperlane/registry
 COPY hyperlane/scripts /home/hyperlane/scripts
+COPY testdata/vkeys /home/hyperlane/testdata/vkeys
```

Comment:
might make more sense to mount this as a docker volume to avoid needing to rebuild the image all the time